serverlessworkflow/README.adoc

```diff
@@ -0,0 +1,56 @@
+= Kogito Serverless Workflow documentation
+
+== Prerequisite
+
+Running Antora with Lunr extension (required to generate self-hosted search) requires Node.js v16.0.0 or above.
+
+== Installing Antora and Lunr
+
+Install Antora CLI 3.0.0 within the playbook project so you can run the antora command using npx:
+
+`npm i -D -E @antora/cli@3.0.0`
+
+Next, install the site generator package within the playbook project:
+
+`npm i -D -E @antora/site-generator@3.0.0`
+
+Now when you run `npx antora -v`, you should see the version of both the Antora CLI and the site generator printed in the terminal.
+
+```
+@antora/cli: 3.0.0
+@antora/site-generator: 3.0.0
+```
+
+Finally install the Lunr extension with:
+
+`npm i @antora/lunr-extension`
```

Comment:
I think this installation recipe may become much simpler if you materialize the dependencies in `package.json`. 

See  https://github.com/kiegroup/optaplanner-website/blob/main/optaplanner-website-docs/package.json.

Reply:
Thanks, I agree using `package.json` would be a clean way.